source/manual/covid-19-services.html.md

```diff
@@ -38,17 +32,32 @@ message from the NHS, or been advised by their GP to fill in the form.
 They can fill the form in themselves, or someone else can fill it in
 for them.
 
+There is also an interactive voice response (IVR) automated phone
+service provided by AWS. This is for users who do not have access to
+the internet.
+
 - [Start page](https://www.gov.uk/coronavirus-extremely-vulnerable)
 - [Service](https://coronavirus-vulnerable-people.service.gov.uk/live-in-england)
 - [GitHub](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form)
 
+## Find support if you're affected by coronavirus
+
+This service allows all citizens - who may not have been eligible for
+the extremely vulnerable service - to find information about what help
+is available if they're struggling with unemployment, an inability to
+get food, having somewhere to live, or their general wellbeing as a
+result of coronavirus.
+
+- [Start page](https://www.gov.uk/find-coronavirus-support)
+- [Service](https://find-coronavirus-support.service.gov.uk/urgent-medical-help)
+- [GitHub](https://github.com/alphagov/govuk-coronavirus-find-support)
 
 ## Architecture
 
 Both applications have a similar architecture of:
 
 - **Framework:** Ruby on Rails
-- **Database:** AWS RDS (business volunteering) and DynamoDB (vulnerable people)
+- **Database:** AWS RDS (business volunteering, find support) and DynamoDB (vulnerable people)
```

Comment:
Isn't the find support page using postgres?

Reply:
RDS is AWS's Postgres/MySQL/etc solution

Reply:
👍